src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,37 @@
+package tests;
+
+import driver.DriverSingleton;
+import helpers.ConfigurationManager;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Listeners;
+import util.TestListener;
+
+import java.io.IOException;
+
+@Listeners({TestListener.class})
+public class BaseTest {
+    protected static WebDriver driver;
+
+    @BeforeMethod
+    public void setUp()  {
+        driver = DriverSingleton.getDriver();
+        driver.manage().window().maximize();
+    }
+
+    @BeforeSuite
+    public void readProperties() throws IOException{
+        ConfigurationManager.loadProperties();
```

Comment:
there is a compilation issue because no ConfigurationManager class in the helpers package